content/blog/2020/10/VSCode-LanguageSupport-0.0.28/index.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "New release of VS Code Language Support for Apache Camel 0.0.28"
+date: 2020-10-27
+authors: [apupier]
+categories: ["Releases","Tooling"]
+preview: "New release of VS Code Language Support for Apache Camel 0.0.28: Camel 3.6.0, Camel Kafka Connector and Camel K Modeline and outline"
+---
+
+A new release of [VS Code Language Support for Apache Camel](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-apache-camel) 0.0.28 is available. It is providing an interesting set of new features.
+
+# Camel 3.6.0 upgrade
+
+Camel [3.6.0](https://camel.apache.org/blog/2020/10/Camel36-Whatsnew/) catalog is now provided by default. The 3.6.0 version is also leveraged internally.
+
+# Camel Kafka Connector Properties file improvements
+
+Previously, the support for [Camel Kafka Connector](https://camel.apache.org/camel-kafka-connector/latest/index.html) was limited to Camel URI in `camel.(sink|source).url` property value.
+This release focused to provide support to the usage of list of properties for each parameters instead of using a potentially long Camel URI. It relies on the new Camel Kafka Connector Catalog.
+
+#### Completion for `connector.class` property value
+
+There is completion for the `connector.class` property value. The display name of the completion item is the simple class name. The fully qualified name is inserted.
+
+![Completion for connector.class](completionConnectorClass.gif "Completion for connector.class") 
+
+#### Completion and hover for component properties
+ 
+There is completion and hover for `camel.sink.*` and `camel.source.*` property keys based on Camel Kafka Connector Catalog.
+  
+![Completion and hover for component properties](completionAndHoverForCamelSinkSourceProperties.gif "Completion and hover for component properties") 
+  
+#### Completion for Converters
+
+Camel Kafka Connector is providing specific Kafka [key](https://kafka.apache.org/documentation/#key.converter) and [value](https://kafka.apache.org/documentation/#value.converter) Converters for some connectors. In this case, they are proposed for completion as property value of `key.converter` and `value.converter`.
```

Comment:
Maybe change "is providing" to "provides"? There are a few places that might be made a bit smoother with that change of phrase.